stetho/src/main/java/com/facebook/stetho/inspector/protocol/module/CSS.java

```diff
@@ -21,7 +24,13 @@
 import org.json.JSONObject;
 
 public class CSS implements ChromeDevtoolsDomain {
-  public CSS() {
+  private final ChromePeerManager mPeerManager;
+  private final Document mDocument;
+
+  public CSS(Document document) {
```

Comment:
since `CSS` isn't actually using `Document` yet, let's not modify `CSS` in this PR